TSC-Charter.md

```diff
@@ -176,10 +187,43 @@ discussion will continue.
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
+While the results of all votes must be made public, the actual individual
+ballots cast for most votes may be made public or confidential at the discretion
+of the TSC Chair. However, individual ballots for recertification or removal of
+TSC members must remain confidential. The names of members voting or abstaining
+in all votes must be made public with a clear indication given of whether the
+individual voted or abstained.
+
+Following the completion of all votes, a public statement must be made via
+GitHub specifying the results of the vote. Contextual detail about why the
+vote was held, including a listing of the specific questions voted on must be
+included in the statement.
+
 Note that, in addition to requiring a simple majority vote of the TSC, all
 changes to this charter are also subject to approval from the Node.js
 Foundation board.
 
+### Votes to Remove Members
+
+Should a situation arise that considers the removal of a TSC member, an
+attempt should first be made to resolve the issue without requiring a vote.
+The TSC Chair may, at their discretion, request that the issue be referred
+to a Foundation appointed third-party mediator for arbitration.
+
+Failing all other possible resolutions, a simple majority vote of TSC members
+is required to remove a member. Should such a vote to result in a tie (50%/50%
```

Comment:
s/vote to result/vote result

Reply:
Should this be 66%/33% or do we like the potential almost 50% non-support?